CONTRIBUTING.md

```diff
@@ -51,18 +51,39 @@ by first clicking the fork button on top of the repository and cloning your fork
 
 ### Testing components locally
 
-If you're hacking on a component and want to test it in a local project, use `npm link`. For example:
+#### Testing in the documentation site
+
+If you're hacking on a component and want to test it in the documentation site follow the following steps.
+
+In this repo:
+- check out a working branch
+- run `npm run cf-link`
+
+In a site specific clone of this repo:
```

Comment:
Suggestion to make this slightly more explicit: "In a second local clone of this repo in a different folder:"